app/assets/stylesheets/components/common/_download_link.scss

```diff
@@ -0,0 +1,15 @@
+// For links that contain download attribute
+
+.download-link {
+
+}
```

Comment:
I don't think this empty declaration is a requirement of BEM - unless you're planning on adding any default block styles to `.download-link` I think this should be removed?